SayTheirNames/Source/Controller/Home/Person/Views/PersonHashtagTableViewCell.swift

```diff
@@ -54,6 +54,7 @@ class PersonHashtagTableViewCell: UITableViewCell {
     }()
     
     private var identifier: String = ""
+    public var person: Person!
```

Comment:
I think it would be more efficient to only store the Hashtags as opposed to the whole `Person` object. I also think it would be nice to initialize with an empty array instead of using a `!`
```suggestion
    public var hashtags = [Hashtag]()
```

Reply:
@Kilo-Loco Nice catch 💯